### DISCUSSION TITLE
Membrane Reed Study

### DISCUSSION THREAD

**Discussion by CamStLouis** [edwardkort/WWIDesigner#143] [Reed Study] [answered]
I've been developing some extreme-low-maintenance pipes designed to be made and played at home by folks with minimal craft experience. I've had good results following some of Linsey Pollack's tutorials and am making 3D printed versions that look more like ordinary pipes than the 'found object' vibe he's famous for. 

Using the reed study model, I think a membrane reed is most like a "lip reed" but I'm curious by what criteria WwID is separating single reed, double reed, and lip reed. 

Thanks as always for maintaining this incredible tool. When I get into conical bore pipes I am stoked to see what insights WwID might put forth!

As always, my open-source designs and experiments may be found here in mesh and .STEP form: https://drive.google.com/drive/folders/1ki-lHh_eoJHVwyg0ekMLZJh7u88CcNcy?usp=sharing

**Comment by bhp1** [comment, marked as answer]
In practice, the single- and double-reed models should give identical results.  There is a slight difference between those, where positive external pressure forces the reed closed, and lip reeds, where positive external pressure forces the "reed" open: single and double reeds will have a positive value for alpha, while we expect lip reeds to have a negative value for alpha.  I suspect you are correct that a membrane reed will have a negative value of alpha, and is best modelled as a lip reed.

**Comment by merrico3216** [comment]
Hello,
are there any new development on the Membrane Reed?
Thanks.

**Reply by CamStLouis** [reply]
Not really. I have posted the latest version on my Thangs page here, but I've just been too busy with other projects. https://thangs.com/designer/Barter%20Loch%20Woodwinds/3d-model/Membrane%20Reed%20Bagpipe%20Chanter-74520

Like other reeds, it's very sensitive to tension and bore-to-length ratio; unlike other reeds, there's almost zero published information to compare to.
